From b4aa95b2151c40b57ebfbe9723e3941bddfa7464 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Jirman?= <megi@xff.cz>
Date: Sun, 21 Nov 2021 17:01:48 +0100
Subject: [PATCH 170/406] arm64: dts: rk3399-pinebook-pro: Remove redundant
 pinctrl-* properties from edp

These are already present in rk3399.dtsi.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
index 1f95578425b5..1f7abb0cc275 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
@@ -416,8 +416,6 @@ &cdn_dp {
 
 &edp {
 	force-hpd;
-	pinctrl-names = "default";
-	pinctrl-0 = <&edp_hpd>;
 	status = "okay";
 
 	ports {
-- 
2.34.1

